### DISCUSSION TITLE
Generte the images in _astro

### DISCUSSION THREAD

**Discussion by wildfiremedia** [withastro/roadmap#1077] [Proposal]
### Body

Currently, when using `<Image />` and compile on macOS, my site (SSR) will be deploy on Linux environment which may have low hardware resources. We could generate multiple image files without constantly relying on Sharp service for on-the-fly image transformation.

**Comment by ematipico** [comment]
Astro offers the API to create an external image service: https://docs.astro.build/en/reference/image-service-reference/#building-using-the-image-services-api
